test.sh

```diff
@@ -108,7 +108,7 @@ function reset_containers() {
 function reset_container() {
     local CONTAINER_TO_BE_RESETTED=$1
     docker_compose stop -t 0 ${CONTAINER_TO_BE_RESETTED} &>> ${LOGFILE};
-    docker_compose rm -vf ${CONTAINER_TO_BE_RESETTED} &>> ${LOGFILE};
+    docker_compose rm -f ${CONTAINER_TO_BE_RESETTED} &>> ${LOGFILE};
```

Comment:
If we don't rely on volumes for the caching, this change isn't needed for the stated purpose of this pull request. (Maybe it's a good idea anyway, though.)